temporalio/lib/temporalio/client.rb

```diff
@@ -478,15 +478,17 @@ def signal_with_start_workflow(
     # List workflows.
     #
     # @param query [String, nil] A Temporal visibility list filter.
+    # @param next_page_token [String, nil] Token for the next page of results. If not set, the first page is returned.
+    # @param page_size [Integer, nil] Maximum number of results to return. If not set, all results are returned.
     # @param rpc_options [RPCOptions, nil] Advanced RPC options.
     #
     # @return [Enumerator<WorkflowExecution>] Enumerable workflow executions.
```

Comment:
Not sure how to express here that a `next_page_token` and `next_page` methods might be available